simplq/src/components/common/QueueHistory/QHistory.css

```diff
@@ -0,0 +1,48 @@
+.history-row {
+    padding: 5px;
```

Comment:
Best to convert this to an scss file and use rem instead of px for sizing wherever possible.